guides/common/modules/proc_updating-flatpak-applications.adoc

```diff
@@ -0,0 +1,24 @@
+[id="updating-flatpak-applications_{context}"]
```

Comment:
This is not specific to Foreman/Satellite, is it? These are generic Flatpak commands and as such, I don't think we should be documenting them in our docs set. Is there some other documentation that we can link to for more details on Flatpak? That would enable you to drop this module entirely.

Reply:
I've asked @sjha4 about this for clarification.

Reply:
I'll refer to this section in the Installing Flatpak applications chapter.